shared/src/balancer/event_handler.rs

```diff
@@ -355,17 +130,20 @@ impl EventStoring<WeightedPool2TokensFactoryEvent> for PoolStorage {
         events: Vec<EthContractEvent<WeightedPool2TokensFactoryEvent>>,
         range: RangeInclusive<BlockNumber>,
     ) -> Result<()> {
-        let balancer_events = convert_two_token_pool_created(events)?;
-        PoolStorage::replace_events_inner(self, range.start().to_u64(), balancer_events).await?;
-        Ok(())
+        PoolStorage::replace_events_inner(
+            self,
```

Comment:
I missed this before but this can likely be written as `self.replace_events_inner(...)`.

Reply:
Yea probably, will check it out.